### DISCUSSION TITLE
Enhancement Request - Parked Vehicles

### DISCUSSION THREAD

**Discussion by rarygit** [UMEP-dev/UMEP#233] [Ideas]
SOLWEIG land cover for paved surfaces excludes vehicles parked in carparks, which can be a significant contributor to land surface temperatures (see FLIR images below). Especially areas covering large carparks servicing shopping centres.

Usually I remove all vehicles from the LiDAR data prior to converting it to raster grids.

In these circumstances, as a feature request, is it possible to add parked vehicles as a component of the Ground land cover, with corresponding emissivity? At certain times of day, especially midday during the summer vacation, the carparks can be almost entirely filled with vehicles.

![FLIR0031_22Aug2019_1440](https://user-images.githubusercontent.com/16574504/107611510-6ef41d00-6c44-11eb-9908-cd8a95e06f68.jpg)
![FLIR0033_22Aug2019_1440](https://user-images.githubusercontent.com/16574504/107611511-6f8cb380-6c44-11eb-9af9-1712d1585608.jpg)

**Comment by biglimp** [comment]
This is actually already possible. Below you find steps to accomplish this. 

1. Create a ground cover where you give car pixels a unique value, e.g. 8 (1 through 7 is taken).
2. Locate **landcoverclasses_2016a.txt** in the UMEP plugin (.\UMEP\SOLWEIG)
3. Add a new line for you car land cover and assign the appropriate values. The three last is the most important here since they are parameters that builds up the daytime surface temperature wave. _Ts_deg_ is the increase in degC for each deg sun altitude during clear weather, _T_start_ is the difference between Tair and Tsurface at sun rise and _TmaxLST_ is the time when Tsurface peaks. See Lindberg et al ([2008](https://link.springer.com/article/10.1007%2Fs00484-016-1135-x)) and ([2016](https://link.springer.com/article/10.1007/s00484-008-0162-7)) for detailed description and how you can set these parameters. Preferably one should have an empirical dataset but you can also set these numbers from experience/knowledge on how Tsurface behaves on cars surfaces.
4. Now you can re-run SOLWEIG with the new ground cover.

One should remember that SOLWEIG calculates GROUND surface temperature (out-going longwave radiation). Since a car is not ground but an object, the correct way might be to estimate Ts underneath the car. However, the model can definitely by exploited in this maner, to derive spatial patterns of outgoing radiation from the surface including car surface temperature. One should though not consider the mean radiant temperature when including cars like this since people usually stay besides the cars, not on top of them...

**Reply by rarygit** [reply]
Thanks for replying! I hadn't seen an alert for your comment. 
This is now a good start to try this out.

I had been reading your 2008 and 2016 articles, but the ground temperature approach you described above was not that clear to me in the articles.

Deriving spatial patterns of the outgoing radiation is my focus, and its mitigation using the soil model.

In the meantime I had been reading the literature on vehicle thermal regime and parameters.

I have attached three, which were interesting from the spatio-temporal dynamics:

Levinson et al. 2011 (Potential benefits of solar reflective car shells: Cooler cabins, fuel savings and emission reductions).

Zhang and Liu, 2016, (Investigation on the Temporal Surface Thermal Conditions for Thermal Comfort Researches Inside A Vehicle Cabin Under Summer Season Climate)

Sunawar et al., 2019 (The characteristics of heat inside a parked car as energy source for thermoelectric generators).
https://link.springer.com/content/pdf/10.1007/s40095-019-0311-2.pdf

From the perspective of SOLWEIG modelling, Sunawar et al. 2019 made two observations:

 1. "In the 20-min test, the temperature outside dropped, because clouds blocked the sun, but the (high) temperature inside the car remained the same and then decreased following the temperature outside."
 
2. "The temperature under the car was stable and almost the same as the ambient one."

That means we could set Ts underneath the car as Tair (ambient temperature).

Zhang and Liu provide a useful sketch of the heat transfer process and corresponding flowchart. They observed that "the external surface is also cooled convectively by the ambient air. Thus, the internal door surface is cooled conductively by the external surface, and its net conduction heat flux decreases and becomes negative." 

In view of these observations, I would anticipate that the external surface temperature of the vehicle remains warmer for an extended period due to the conduction of the internal heat.


![Fig1_Heat Transfer Process of Vehicle](https://user-images.githubusercontent.com/16574504/109616780-dca1b380-7b35-11eb-8928-e1911dbef592.jpg)

[000_2011.Levinson_Potential_benefits_of_solar_reflective_car_shells.pdf](https://github.com/UMEP-dev/UMEP/files/6067043/000_2011.Levinson_Potential_benefits_of_solar_reflective_car_shells.pdf)
[000_2016.Zhang_Temporal Surface Thermal Conditions Inside A Vehicle Cabin in Summer.pdf](https://github.com/UMEP-dev/UMEP/files/6067045/000_2016.Zhang_Temporal.Surface.Thermal.Conditions.Inside.A.Vehicle.Cabin.in.Summer.pdf)
[000_2019.Sunawar_Characteristics of heat inside a parked car.pdf](https://github.com/UMEP-dev/UMEP/files/6067046/000_2019.Sunawar_Characteristics.of.heat.inside.a.parked.car.pdf)
